salt/metalk8s/addons/dex/deployed/login.sls.in

```diff
@@ -0,0 +1,14 @@
+#!jinja | metalk8s_kubernetes kubeconfig=/etc/kubernetes/admin.conf&context=kubernetes-admin@kubernetes
+
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: dex-login
```

Comment:
Let's call this file `theme-configmap.sls` for clarity, or just `configmap` if you prefer. Login is too vague IMHO.